scripts/build_biomass_potentials.py

```diff
@@ -211,15 +251,103 @@ def convert_nuts2_to_regions(bio_nuts2, regions):
     return bio_regions
 
 
+def add_unsustainable_potentials(df):
+    """
+    Add unsustainable biomass potentials to the given dataframe. The difference
+    between the data of JRC and Eurostat is assumed to be unsustainable
+    biomass.
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+        The dataframe with sustainable biomass potentials.
+    unsustainable_biomass : str
+        Path to the file with unsustainable biomass potentials.
+
+    Returns
+    -------
+    pd.DataFrame
+        The dataframe with added unsustainable biomass potentials.
+    """
+    if "GB" in snakemake.config["countries"]:
+        latest_year = 2019
+    else:
+        latest_year = 2021
+    idees_rename = {"GR": "EL", "GB": "UK"}
+    df_unsustainable = (
+        build_eurostat(
+            countries=snakemake.config["countries"],
+            input_eurostat=snakemake.input.eurostat,
+        )
```

Comment:
For performance:

use `build_eurostat(..., nprocesses=int(snakemake.threads))`

and add to rule specification of `build_biomass_potentials`:

```
threads: 8
```